advocacy_docs/partner_docs/Cyberark/05-Usingcyberark.mdx

```diff
@@ -0,0 +1,109 @@
+---
+title: 'Using'
+description: 'Walkthrough of example usage scenarios'
+---
+
+After you have configured your EDB Postgres Advanced Server and CyberArk, as stated in the Configuring section, you will be able to then use CyberArk (CPM Central Policy Manager) functionality. 
+
+Authorized users can change passwords that are stored in the Safe through the PVWA (Password Vault Web Access). These passwords can be changed manually or replaced by a new password that is randomly generated by the Central Policy Manager. The CPM generates unique and highly secure passwords using the password policy and the random password generation mechanism. 
+
+The following example demonstrate generating unique and highly secure passwords based on the password policy by CPM (CPM Central Policy Manager).
+
+- Configure Password Policy for PostgreSQL Platform
+  - Configure password policy by setting the Password Management parameters for PostgreSQL platform
+- Create the Account for EDB Postgres Advanced Server running instance
+  - Setup Account on PVWA (Password Vault Web Access) for the EDB Postgres Advanced Server running instance, the password will be managed for this EDB Postgres Advanced Server running instance
+- Change the Password 
+  - Using CPM (CPM Central Policy Manager) change the password for the EDB Postgres Advanced Server running instance
+- Verify that User can login to EDB Postgres Advanced Server
+  - After the password is changed by CPM (CPM Central Policy Manager), login to EDB Postgres Advanced Server with the changed password.
+
+## Configure Password Policy for PostgreSQL Platform
+
+The following steps demonstrate the Configuration of password policy by setting the Password Management parameters for PostgreSQL platform.
+
+1. Login to PVWA (Password Vault Web Access).
+
+2. Click Administration option, and then select Platform Management. A list of supported target account platforms appears.
+
+3. Select Databases —> PostgreSQL, and then click Edit. The platform parameters appear.
+
+![Select Platform](Images/platfom.png)
+
+4. In the Automatic Password Management —> General parameters Section, make the following change.
+       
+       - Set the ImmediateInterval parameter to 1. (The number of minutes that will elapse between when the user initiates an account management process and when the process is performed.)
```

Comment:
is the Immediate Interval meant to be one word?

Reply:
Maybe if so we can add the `` around it so it formats differently.

Reply:
done